### ISSUE TITLE
Supporting URL deduplication

### ISSUE THREAD

**Issue by Regala** [simioni87/auth_analyzer#17]
Hi @simioni87 !

One thing that happens very frequently when doing access control testing is testing the same URLs over and over again. It would be super helpful if the extension supported a very simple URL de duplication feature; so, if ON, a request that has already been processed by Auth Analyzer (e.g. it's already on the results table) would be ignored. 

Even a very basic support of this - e.g. really just comparing a GET request if it's exactly the same (not taking in consideration ?parameters for example) would be a huge help.

Thanks!

**Comment by simioni87** [comment]
Hi regala. 

Thanks for the input, worth to take it into consideration for the next release from my point of view.

For now, you at at least able to hide duplicates at the table. Means you will only see the first repeated request if any duplicates are present. Have you seen this filter?

BR

**Comment by Regala** [comment]
Oh, so if you uncheck the duplicates checkbox it only show uniques?

**Comment by simioni87** [comment]
Exactly. However, duplicates are still repeated in the background; the are just not shown in the table...

**Comment by Regala** [comment]
Gotcha! Ok that's already super good. The additional check could help just
to save resources but agree is not essential then, just a cool optimization
:)
